sympy/simplify/sqrtdenest.py

```diff
@@ -236,6 +236,18 @@ def _sqrtdenest0(expr):
         else:
             n, d = [_sqrtdenest0(i) for i in (n, d)]
             return n/d
+
+    if isinstance(expr, Add):
+        cs = []
+        args = []
+        for arg in ordered(expr.args):
```

Comment:
The arguments of an Add are ordered canonically when the node is created. What is the purpose of this `ordered`?